src/views/InteractiveMap/InteractiveMap.vue

```diff
@@ -427,6 +453,74 @@ export default Vue.extend({
           }
         });
     },
+    simulateDiffFVACard(card, model) {
+      const operations = this.cardModifications(card);
+      this.updateCard({
+        uuid: card.uuid,
+        props: {
+          isSimulating: true,
+          hasSimulationError: false
+        }
+      });
+      axios
+        .post(`${settings.apis.model}/simulate`, {
+          model_id: model.id,
+          method: card.method,
+          operations: operations,
+          objective_id: card.objective.reaction
+            ? card.objective.reaction.id
+            : model.default_biomass_reaction,
+          objective_direction: card.objective.maximize ? "max" : "min"
+        })
+        .then(response => {
+          const wildtypeGrowthrate = response.data.growth_rate;
+          const wildtypeFluxdistribution = response.data.flux_distribution;
+          // Calculate new bounds based on manipulation scores calculated by DiffFVA
+          // (Reference flux * score) / Reference growth
+          const editedBounds = card.manipulations
+            .map(function(manipulation) {
+              const newBound = Math.round(
+                (wildtypeFluxdistribution[manipulation.id] *
+                  manipulation.value) /
+                  wildtypeGrowthrate
+              );
+              return {
+                operation: "modify",
+                type: "reaction",
+                id: manipulation.id,
+                data: {
+                  lower_bound: newBound,
+                  upper_bound: newBound
+                }
+              };
+            })
+            .filter(function(operation) {
+              if (operation.data.lower_bound === 0) {
+                return false;
+              }
+              return true;
+            });
+          // Re-simulate the model with the updated bounds
+          // after readding all previous modifications
+          const filteredMods = this.cardModifications(card).filter(
+            mod => mod.operation !== "modify"
```

Comment:
Maybe I missed something - why is it necessary to remove existing modifications? Does DiffFVA return constraints at all? Note also that all modifications are applied in order, so if a reaction is modified twice, the latest operation will take effect. So you could consider just adding the DiffFVA constraints last.

Reply:
> Note also that all modifications are applied in order, so if a reaction is modified twice, the latest operation will take effect. So you could consider just adding the DiffFVA constraints last.

This I didn't know. I fixed it now!